app/grandchallenge/algorithms/forms.py

```diff
@@ -93,16 +94,27 @@ class JobCreateForm(SaveFormInitMixin, Form):
     algorithm_model = ModelChoiceField(
         queryset=None, disabled=True, required=False, widget=HiddenInput
     )
+    creator = ModelChoiceField(
+        queryset=None, disabled=True, required=False, widget=HiddenInput
+    )
+    time_limit = IntegerField(
+        disabled=True, required=False, widget=HiddenInput
+    )
```

Comment:
Is there a test to ensure that the `time_limit` cannot be altered by the user? There is no other guard other than using `HiddenInput`.